### ISSUE TITLE
Increase the Max Call forwarding number length to 16.

### ISSUE THREAD

**Issue by v01ded** [fusionpbx/fusionpbx#2708]
Hi,

Currently, the maxmium length of call forwarding number is 11. I would like to suggest to increase the length to 16 char.

Stan

**Comment by markjcrane** [comment]
There are many ways to call forward please

**Comment by markjcrane** [comment]
This was put in place to prevent forwarding to international numbers. It was an intentional security precaution.

**Comment by mafoo** [comment]
maybe this is something that could be set via default/domain setting so by default it is on 11 but it allows an admin to override it (with the description set to a warning)

**Comment by moteus** [comment]
toll_allow works for call forward so is it really need this limitation?

**Comment by v01ded** [comment]
My setup is used by colleagues in various countries and have may Trunk from various country connect. As there are overlaying area code in these countries, i have to enforce international dialing for all outgoing calls. Therefore call forwarding using international numbers(more than 11 char) is needed.

**Comment by markjcrane** [comment]
I agree with mafoo a setting to control this would be best. I tend to lean to secure by default with option to adjust settings.

**Comment by markjcrane** [comment]
toll_allow can help but not everyone is using it so relying entirely on it entirly seems risky.
